### DISCUSSION TITLE
Is this extension compatible with the Official Prettier PHP Plugin?

### DISCUSSION THREAD

**Discussion by trymeouteh** [lkrms/pretty-php#123] [Q&A]
Is this extension compatible with the Official Prettier PHP Plugin?

https://github.com/prettier/plugin-php

If I used this extension in my editor while another developer using a different editor on their computer and uses the Official Prettier PHP Plugin, would be both have the same formatted PHP code?

**Comment by lkrms** [comment]
`pretty-php` (written in PHP to leverage its built-in parser and simplify use in CI workflows) doesn't have anything to do with the Prettier PHP plugin (written in JS), so its output isn't the same. I haven't experimented with Prettier recently enough to comment on how similar it is, but I would expect results to vary significantly in some cases and not at all in others, depending on the input.

Which editors are you and your colleagues using?

**Reply by trymeouteh** [reply]
I am using VSCodium (FOSS version of VSCode)

**Reply by lkrms** [reply]
> I am using VSCodium (FOSS version of VSCode)

I've used VSCodium myself. The `pretty-php` extension is maintained on the Open VSX registry for FOSS users: https://open-vsx.org/extension/lkrms/pretty-php

At this stage there aren't any other integrations, but I'm hoping to be able to do something about that soon!
